scaledimage.cpp

```diff
@@ -13,6 +14,11 @@ ScaledImage::ScaledImage(std::string filename, int rotateMode) {
 		return;
 	}
 	
+	if (tmpload->w==SCREEN_WIDTH && tmpload->h<=SCREEN_HEIGHT || tmpload->h==SCREEN_HEIGHT && tmpload->w<=SCREEN_WIDTH)

```

Comment:
The condition seems superfluous to me, isn't the second part enough?

Reply:
indeed, removed 2nd part but left if statement for readability